lib/netext/httpext/response.go

```diff
@@ -158,11 +169,38 @@ func (res *Response) JSON(selector ...string) (interface{}, error) {
 		}
 
 		if err := json.Unmarshal(body, &v); err != nil {
+			switch t := err.(type) {
+			case *json.SyntaxError:
+				err = checkErrorInJSON(body, int(t.Offset), err)
+			default:
+				break
+			}
```

Comment:
I think this will be better as :
```
if syntaxError, ok := err.(*json.SyntaxError); ok {
   err = checkErrorInJSON(body, int(syntaxError.Offset), err)
}
```
Much more clean and I technically highly doubt we will ever handle any other error in this case :)